serverless/src/app.js

```diff
@@ -45,36 +46,28 @@ export default async (event) => {
     delete body.STELLAR_NETWORK
     delete body.txFunction
 
-    // const txFunction = new Function(`
-    //   __filename,
-    //   __dirname,
-    //   require,
-    //   module,
-    //   HORIZON_URL,
-    //   STELLAR_NETWORK,
-    //   global,
-    //   globalThis,
-    //   process
-    // `, // leave [global, globalThis, process] empty to effectively unset them
-    // `
-    //   'use strict'; 
-    //   ${txFunctionCode}; 
-    //   return module.exports;
-    // `)(
-    //   __filename,
-    //   __dirname,
-    //   require,
-    //   {exports: null},
-    //   HORIZON_URL,
-    //   STELLAR_NETWORK,
-    // )
-    // const result = await txFunction(body)
+    const vm = new NodeVM({
+      console: 'off',
+      eval: false,
+      wasm: false,
+      strict: true,
+      fixAsync: true,
+      sandbox: {
+        HORIZON_URL,
+        STELLAR_NETWORK
+      },
+      require: {
+        builtin: null,
```

Comment:
It think we need to support a handful of builtin functions. 

Especially when reading from `AccountData` node programs have to make use of `Buffer`

We should support `buffer` as least. A few others that don't seem harmful are `utils`, `url`, `crypto`, and `timers`.

Reply:
These should all be allowed by default thanks to NodeVM
https://github.com/patriksimek/vm2#nodevm

e.g. https://stackblitz.com/edit/node-ecdivm?devtoolsheight=33&file=index.js